spec/controllers/follows_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```

Comment:
Yeah, I wrote a controller test because I couldn't write a failing feature test. OTOH, now I've managed to make a feature test fail, I can rip this out. Thanks for the reminder!